chart/templates/mayastor/io/io-engine-daemonset.yaml

```diff
@@ -110,7 +110,8 @@ spec:
         - "--env-context=--{{ .Values.io_engine.envcontext }}"{{ end }}{{ if .Values.io_engine.reactorFreezeDetection.enabled }}
         - "--reactor-freeze-detection"{{ end }}
         - "--tgt-crdt={{ .Values.io_engine.target.nvmf.hostCmdRetryDelay.crdt1 }}"{{ if .Values.eventing.enabled }}
-        - "--events-url=nats://{{ .Release.Name }}-nats:4222"{{ end }}
+        - "--events-url=nats://{{ .Release.Name }}-nats:4222"{{ end }}{{ if .Values.io_engine.pstorRetries}}
+        - "--ps-retries={{ .Values.io_engine.pstorRetries }}"{{ end }}
```

Comment:
Should we use default rather than if? For upgrades with old values?

Reply:
Yeah we can? Should I do that now?

Reply:
This is new value, this should get pulled in. Upgrades via upgrade job should be fine.

Reply:
Ohh then we don't need it? I have added it anyway. Should I remove it?

Reply:
It's fine either way. Don't need to change IMO.